pkg/command/service/common.go

```diff
@@ -144,3 +145,17 @@ func GetIngressController(p *pkg.PerfParams) map[string]string {
 	ingressController["version"] = "Unknown"
 	return ingressController
 }
+
+// deleteFile deletes a file from the filepath
+func deleteFile(filepath string) error {
+	_, fileError := os.Stat(filepath)
```

Comment:
We tend to just use `err` for error name but cool :)

Reply:
Thank you, I will change error name to `err` in new version.